src/styles/global.css

```diff
@@ -13,6 +13,10 @@ body, * {
     font-family: 'Lato', sans-serif;
 }
 
+body *:focus {
+    outline: dashed 0.25rem #993404;
```

Comment:
This makes a consistent global focus behavior using a dashed orange outline, which is more obvious than Chrome's default.